src/materials/Material.js

```diff
@@ -479,6 +479,21 @@ class Material extends EventDispatcher {
 
 	}
 
+	// @deprecated since r131, f5803c62cc4a29d90744e9dc7811d086e354c1d8
+
+	get vertexTangents() {
+
+		console.warn( 'THREE.' + this.type + ': .vertexTangents has been removed.' );
+		return false;
+
+	}
+
+	set vertexTangents( value ) {
+
+		console.warn( 'THREE.' + this.type + ': .vertexTangents has been removed.' );
+
+	}
+
```

Comment:
Considering that we're in the r141 cycle already, I guess we can remove this one too?

Reply:
It would actually be better to keep such things from rN release to rN+10 release to allow upgrading like r70-r80-r90-r100-... instead of r70-r79-r88-r97-...

Reply:
Ah, so then the legacy code stays for 11 releases. Yeah, that sounds good